### PR TITLE
Added changes for 3.9.x

### PR DESCRIPTION
**Pull request by Jonsy13**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: charts/litmus/templates/auth-server-deployment.yaml [hunk 1]
```diff
@@ -122,18 +121,48 @@ spec:
...
+            - name: ENABLE_INTERNAL_TLS
+              value: "true"
+            - name: TLS_CERT_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/tls.crt"
+            - name: TLS_KEY_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/tls.key"
+            - name: CA_CERT_TLS_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/ca.crt"
+              {{- else }}
+            - name: ENABLE_INTERNAL_TLS
```
Comment (ispeakc0de) [inline charts/litmus/templates/auth-server-deployment.yaml:136]: this env is duplicate

File: charts/litmus/templates/auth-server-deployment.yaml [hunk 2]
```diff
@@ -122,18 +121,48 @@ spec:
...
+            - name: "LITMUS_GQL_GRPC_PORT"
+              value: "{{ .Values.portal.server.graphqlServer.service.graphqlRpcServer.port }}"
+              {{- if .Values.internalTLS.enabled }}
+            - name: ENABLE_INTERNAL_TLS
+              value: "true"
+            - name: TLS_CERT_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/tls.crt"
+            - name: TLS_KEY_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/tls.key"
+            - name: CA_CERT_TLS_PATH
```
Comment (ispeakc0de) [inline charts/litmus/templates/auth-server-deployment.yaml:133]: Have we added support for insecure skip secure verify as well?

File: charts/litmus/templates/server-deployment.yaml [hunk 3]
```diff
@@ -131,24 +130,29 @@ spec:
...
+            - name: ENABLE_INTERNAL_TLS
+              value: "true"
+            - name: TLS_CERT_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/tls.crt"
+            - name: TLS_KEY_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/tls.key"
+            - name: CA_CERT_TLS_PATH
+              value: "{{ .Values.internalTLS.certMountPath }}/ca.crt"
+              {{- else }}
+            - name: ENABLE_INTERNAL_TLS
```
Comment (ispeakc0de) [inline charts/litmus/templates/server-deployment.yaml:147]: same as above

**Review by ispeakc0de — APPROVED** [review]

File: charts/litmus/templates/auth-server-deployment.yaml (hunk 1 above)
Comment (Jonsy13) [reply charts/litmus/templates/auth-server-deployment.yaml:136]: One is having value as `false` & one as `true`

File: charts/litmus/templates/server-deployment.yaml (hunk 3 above)
Comment (Jonsy13) [reply charts/litmus/templates/server-deployment.yaml:147]: One is having value as `false` & one as `true`

File: charts/litmus/templates/auth-server-deployment.yaml (hunk 2 above)
Comment (Jonsy13) [reply charts/litmus/templates/auth-server-deployment.yaml:133]: It will be added in upcoming releases.

**Review by Jasstkn — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Pull request merged**